lesson5/src/Zadanie24/Time.java

```diff
@@ -0,0 +1,73 @@
+package Zadanie24;//Создать класс и объекты, описывающие промежуток времени. +
+//        Сам промежуток должен задаваться 3 свойствами: секундами, минутами, часами. +
+//        Сделать методы
+//        * для получения полного количества секунд в объекте, +
+//        * сравнения двух объектов(метод должен работать аналогично compareTo в строках).+
+//        Создать два конструктора:
+//        *получающий общее количество секунд +
+//        * часы, минуты и секунды по отдельности +
+//        Сделать метод для вывода данных+
+
+public class Time {
+
+
+    private long sec;
+    private int min;
+    private int hours;
+
+
+    public Time(long sec, int min, int hours){
```

Comment:
you can put (100000000, 1, 1) on creation. It means that you have 1 hour, 1 minute and a lot of seconds. Make this data appropriate please